### DISCUSSION TITLE
reconnection issue

### DISCUSSION THREAD

**Discussion by bilalbayasut** [kirimwa-id/forum#182] [Tanya Jawab]
Saya pake script dan run setiap jam, di script tersebut saya cek dulu statusnya device nya disconnected apa ga, kalo disconnected maka saya panggil api reconnect. tapi saya sadari bahwa sekitar 24 jam, device menjadi authenticated sehingga saya harus scan manual, bagaimana solusinya ya supaya bisa ga perlu scan qr setiap hari ? thanks @rioastamal

tambahan: waktu trigger reconnect, malah jadi unauthenticated ternyata

**Comment by mmafrizalsp** [comment]
sama juga ini
